lib/platform/common.ts

```diff
@@ -149,6 +149,11 @@ export interface EnsureCommentConfig {
   topic: string;
   content: string;
 }
+export interface EnsureCommentRemovalConfig {
```

Comment:
Could you submit this refactoring as a separate PR that we merge first? Then the change in *functionality* in this PR will be more easily reviewed.